CMakeLists.txt

```diff
@@ -4,18 +4,18 @@ cmake_minimum_required(VERSION 3.16)
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/Modules)
 
 set(${PROJECT_NAME}_MAJOR_VERSION 01)
-set(${PROJECT_NAME}_MINOR_VERSION 06)
+set(${PROJECT_NAME}_MINOR_VERSION 07)
```

Comment:
I increased the minor version because the header files have changed. For a headerless package like the backend this would not have been necessary. It's basically "latest release".

Reply:
That was because it's the version I tested on. I don't know if it works with the older requirements.
As there are no dev-doocs packages available for the older version anyway, it should not matter.